tests/tlv_json_lwm2m_data_test.c

```diff
@@ -384,6 +384,26 @@ static void test_10(void)
     test_data("/12/0", LWM2M_CONTENT_JSON, data1, 17, "10b");
 }
 
+static void test_11(void)
+{
+    const char * buffer = "{\"bn\":\"/5/0/1\",              \
+                         \"e\":[                            \
+                           {\"n\":\"/1\",\"sv\":\"http\"}]  \
+                      }";
+
+    test_raw(NULL, (uint8_t *)buffer, strlen(buffer), LWM2M_CONTENT_JSON, "11");
+}
+
+static void test_12(void)
+{
+    const char * buffer = "{\"bn\":\"/5/0\",                \
+                         \"e\":[                            \
+                           {\"n\":\"/1\",\"sv\":\"http\"}]  \
+                      }";
+
+    test_raw(NULL, (uint8_t *)buffer, strlen(buffer), LWM2M_CONTENT_JSON, "11");
```

Comment:
I think the last argument here should be "12".

Reply:
that's correct, let me make the change